src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -436,6 +436,27 @@ export const SentryApplication = PropTypes.shape({
   status: PropTypes.string,
 });
 
+export const SentryAppInstallation = PropTypes.shape({
```

Comment:
So I started adding sentryTypes as I thought they were needed but I found a workaround by casting the `propTypes` as `any`. But since I already wrote these types, I decided to just leave them in. I can remove them if people think that's a better.

Reply:
If we aren't using them then we shouldn't keep them. Proptypes are on the way out as we convert to TS.

Reply:
@markstory Cool, just got rid of them